docs/api/incoming-message.md

```diff
@@ -80,3 +80,25 @@ An `Integer` indicating the HTTP protocol major version number.
 An `Integer` indicating the HTTP protocol minor version number.
 
 [event-emitter]: https://nodejs.org/api/events.html#events_class_eventemitter
+
+#### `response.rawHeaders`
+
+A `string[]` containing the raw HTTP response headers exactly as they were
+received. The keys and values are in the same list. It is not a list of
+tuples. So, the even-numbered offsets are key values, and the odd-numbered
+offsets are the associated values. Header names are not lowercased, and
+duplicates are not merged.
```

Comment:
This seems like a weird design choice. Why not a list of tuples?

Reply:
Because that's what the equivalent thing in the Node.js HTTP/HTTPS module returns: https://nodejs.org/api/http.html#messagerawheaders

I don't mind doing something else if y'all want to diverge, but it was my understanding that the `net` module was supposed to stay fairly similar to the matching Node.js APIs.

Reply:
huh, weird. okay, I'm fine with aping Node.js here then.